pkg/capabilities/consensus/ocr3/ocr3cap/ocr3cap_data_feeds_consensus-schema.json

```diff
@@ -35,7 +35,8 @@
         "aggregation_method": {
           "type": "string",
           "enum": [
-            "data_feeds"
+            "data_feeds",
+            "reduce"
```

Comment:
@justinkaseman I don't think this is correct -- or at least it's not what we intended for the consensus capability: we expect each aggregation method to have its own config + struct, rather than extending the existing data feeds consensus one.

For identical consensus, I created this struct here: https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/capabilities/consensus/ocr3/ocr3cap/identical_consensus.go, you'll need to create an equivalent for Reduce consensus.

Reply:
Oops, yeah this shouldn't be there. Good catch. I'll make the new schema.

Reply:
Like so? https://github.com/smartcontractkit/chainlink-common/pull/842/commits/2076c0f23f5e9107043d705ad5a5512f4aab82c2

Reply:
Ahh, I see what you mean. Added.